docs/docs/install/configuration.md

```diff
@@ -0,0 +1,34 @@
+| DEBUG                  | A boolean that turns on/off debug mode. Enabling debug mode will show a detailed error page. It should be turned off in production servers to avoid leaking sensitive data. Refer [Django docs](https://docs.djangoproject.com/en/4.2/ref/settings/#debug) for more details.                                                                                                                                                                                                                                                                        | False                        |
+| ALLOWED_HOSTS          | A comma separated list of hostnames that can serve Mathesar. It will be added to Django [ALLOWED_HOSTS](https://docs.djangoproject.com/en/4.2/ref/settings/#allowed-hosts) settings. Add `localhost` if you want to Mathesar to be accessible from localhost too.                                                                                                                                                                                                                                                                                   | `*`                          |
+| DJANGO_DATABASE_KEY    | ***Internal*** The key used for referencing the [Database config information](https://docs.djangoproject.com/en/4.2/ref/settings/#databases). There should be atleast one `default` database config.                                                                                                                                                                                                                                                                                                                                                | `default`                    |
+| DJANGO_DATABASE_URL    | A Postgres connection string of the database which will be used for storing metadata related to Mathesar in the format of: `postgres://{user}:{password}@{hostname}:{port}/{database-name}`. For example, for a database role whose username is `mathesar` and  `mathesar` is the password, `mathesar_db` is the database name and `5432` is the port under which the postgres server is listening on and `mathesar_django` is the database name, then the connection string will be`postgres://mathesar:mathesar@mathesar_db:5432/mathesar_django` | *Required                    |
+| MATHESAR_DATABASES     | A dictionary of database connection string of the external databases to be managed by Mathesar keyed by a unique database key. It should of the format `(<unique_db_key>\|postgres://{user}:{password}@{hostname}:{port}/{database1-name}),(<unique_db_key>\|postgres://{user}:{password}@{hostname}:{port}/{database2-name}),...`.                                                                                                                                                                                                                 | *Required                    |
```

Comment:
There are some grammatical errors. I'd suggest using some tool to help identify them. Perhaps @seancolsen could be of better help identifying/reviewing them. 
Eg.,
>  It should of the format
> It should **be** of the format

Reply:
I am using two tools to check for grammatical errors and it is weird that both didn't catch this particular error.